WordPress/Classes/Models/BasePost.h

```diff
@@ -3,48 +3,50 @@
 #import "DateUtils.h"
 #import "PostContentProvider.h"
 
+NS_ASSUME_NONNULL_BEGIN
+
 typedef enum {
     AbstractPostRemoteStatusPushing,    // Uploading post
     AbstractPostRemoteStatusFailed,      // Upload failed
     AbstractPostRemoteStatusLocal,       // Only local version
     AbstractPostRemoteStatusSync,       // Post uploaded
 } AbstractPostRemoteStatus;
 
-extern NSString * const PostStatusDraft;
-extern NSString * const PostStatusPending;
-extern NSString * const PostStatusPrivate;
-extern NSString * const PostStatusPublish;
-extern NSString * const PostStatusScheduled;
-extern NSString * const PostStatusTrash;
-extern NSString * const PostStatusDeleted;
+extern NSString * _Nonnull const PostStatusDraft;
+extern NSString * _Nonnull const PostStatusPending;
+extern NSString * _Nonnull const PostStatusPrivate;
+extern NSString * _Nonnull const PostStatusPublish;
+extern NSString * _Nonnull const PostStatusScheduled;
+extern NSString * _Nonnull const PostStatusTrash;
+extern NSString * _Nonnull const PostStatusDeleted;
```

Comment:
No need for all the `_Nonnull`s since this is inside a `NS_ASSUME_NONNULL_BEGIN`/`NS_ASSUME_NONNULL_END` block